web-components/src/internals/baseElement/baseElement.ts

```diff
@@ -1,11 +1,14 @@
 import { LitElement, unsafeCSS } from 'lit';
 import global from './global.css?inline';
+import disableAnimations from './disable-animations.css?inline';
 
 export const ActionKeystrokes = [' ', 'Enter'];
 
 export default class BaseElement<EventsPayloadMap = Record<string, never>> extends LitElement {
 
-  protected static globalStyles = unsafeCSS(global);
+  protected static globalStyles = import.meta.env.VITE_IS_CHROMATIC
```

Comment:
i am not the biggest fan of doing this in the baseElement.
Could we do something like this in the .stories files to disable the animations?
```
var sheet = new CSSStyleSheet()
sheet.replaceSync( `.color { color: pink }`)
host.shadowRoot.adoptedStyleSheets.push(sheet) 
```

Reply:
That's a good point! For the reset.css we used this as a base: https://dev.to/hankchizljaw/a-modern-css-reset-6p3
As you can see, in the end it disables all animations for people that have reduced motion set. With our client we had the problem though, that they used virtual desktops which always had reduced motion on, so we never got any animations which led us to remove this CSS.

We then ended up setting it for the bigger animations on a per component base. You can see the loadingPlaceholder for example: https://github.com/Zuehlke/design-system-starter/blob/main/web-components/src/components/loadingPlaceholder/loadingPlaceholder.css